packages/gatsby/src/commands/build-utils.ts

```diff
@@ -66,11 +66,84 @@ export const removePageFiles = async (
   })
 }
 
+// almost c&p from `cache-dir/normalize-page-path
+function normalizePagePath(path: string): string {
+  if (path === `/`) {
+    return `/`
+  }
+  if (path.charAt(path.length - 1) === `/`) {
+    return path.slice(0, -1)
+  }
+  return path
+}
```

Comment:
do we have something like this already in Node.js portion of code? I did few greps and couldn't find one :(